flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobServer.java

```diff
@@ -426,4 +440,11 @@ void unregisterConnection(BlobServerConnection conn) {
 		}
 	}
 
+	/* Secure cookie to authenticate */
+	@Override
+	public String getSecureCookie() { return secureCookie; }
+
+	/* Flag to indicate if service level authentication is enabled or not */
+	public boolean isSecurityEnabled() { return secureCookie != null; }
```

Comment:
I think we can not decide whether security is enabled only based on the secure cookie's value.